src/components/game/GameCard.js

```diff
@@ -101,17 +109,14 @@ const CardBody = ({ article, nextButton }) => {
         onChange={changeHandler("reuters")}
```

Comment:
I _think_ for this one, I had already made a changeHandler that was trying to do two jobs, capturing the radio button change and comparing the sources, which should have been in the submitHandler. This was just how the refactor/separation of the two functions (changeHandler and handleSubmit) played out. But, really, the changeHandler function isn't really even needed, since an anonymous function would work on the onChange, like you mentioned.

I think in my previous edit to the changeHandler, it made sense at the time because I was passing it in 5 different times so it was an attempt to dry up the code. But now, I've got the RadioButtons rendering via a map, so won't be repeating that code anyway. Removing the changeHandler now makes a lot of sense.